pkg/operator/etcdendpointscontroller/etcdendpointscontroller_test.go

```diff
@@ -269,7 +268,7 @@ func TestBootstrapAnnotationRemoval(t *testing.T) {
 				}
 			},
 		},
-		{
+		/* {
```

Comment:
feel free to remove those testcases, we should trust what we get from library-go